atlp-rwanda/ecommerce-app-champions-bn
# ft-register-vendor():

#### What does this PR do?
Register Vendors
documented with swagger
#### Description of Task to be completed

- the administrator should be able to register a new vendor
-  an email containing a random password should be sent to the provided email 
- the vendor should be able to sign in using the provided email and password
- appropriate error messages are displayed on unsuccessful attempts
- the tests are passing
- documented with swagger
#### How should this be manually tested?

- clone the repo
- use npm install to install all packages
- use  npx sequelize db:migrate to migrate migrations
- use npm run dev to start the server
- use the appropriate router in postman or thunderclient  forexample localhost5050/example
- you should receive a random password on the provided email or
- navigate to localhost/5050/docs and test it using the documented API, the result should be the same
- use npm test to test the endpoint
